### ISSUE TITLE
Item Return to Vendor

### ISSUE THREAD

**Issue by maheshstomp** [StompMarket/helpdesk#975]
Need a screen where Seller can mention Items returning to Vendor after no sale or other issue.

**Comment by anban4u** [comment]
Yes. I am planning to add functionality where vendors will be able to track their fulfilled POs, Payments and returns. This will be a part of that
